client/src/components/application/NameExamination.vue

```diff
@@ -3,7 +3,11 @@
   <div>
     <div class="upper-section container-fluid">
 
-      <div class="namePage">
+      <div class="namePage" >
+
+        <div v-bind:style="{ 'margin-left': '-15px', 'margin-right': '-15px', 'margin-top': '-22px', 'background-color': bgColor }">
```

Comment:
a) The styles (other than background colour) should be in a class, not inline style.
b) I think all of this logic should be contained in the StateHeader.vue component and not spread between NameExamination and StateHeader.